imagej.html

```diff
@@ -201,7 +201,7 @@
 <li>
 <p class="first"><a class="anchor" id="screenshot1"></a>Download the latest 5.1.x version of the <span class="bold">Bioformats Package</span> plugin from:</p>
```

Comment:
Outside the scope of this PR I realize but this version number is going to need updating (and all the downloads screenshots updated)

Reply:
I will bump the version in this PR
we can update screenshot in a follow up one